test/parallel/test-net-autoselectfamily-commandline-option.js

```diff
@@ -87,9 +87,8 @@ function createDnsServer(ipv6Addr, ipv4Addr, cb) {
           assert.strictEqual(error.message, `connect ECONNREFUSED ::1:${port}`);
         } else if (error.code === 'EAFNOSUPPORT') {
           assert.strictEqual(error.message, `connect EAFNOSUPPORT ::1:${port} - Local (undefined:undefined)`);
-        } else {
-          assert.strictEqual(error.code, 'EADDRNOTAVAIL');
-          assert.strictEqual(error.message, `connect EADDRNOTAVAIL ::1:${port} - Local (:::0)`);
+        } else if (error.code === 'EADDRNOTAVAIL' || error.code === 'EUNATCH') {
+          assert.strictEqual(error.message, `connect ${error.code} ::1:${port} - Local (:::0)`);
```

Comment:
This doesn't match existing code behavior. Should add a separate else if leg like the EAFNOSUPPORT above.

```suggestion
        } else if (error.code === 'EUNATCH') {
          assert.strictEqual(error.message, `connect EUNATCH ::1:${port} - Local (:::0)`);
        } else {
          assert.strictEqual(error.code, 'EADDRNOTAVAIL');
          assert.strictEqual(error.message, `connect EADDRNOTAVAIL ::1:${port} - Local (:::0)`);
```

Either that or merge the EAFNOSUPPORT else block in to this one. The other tests have this same issue.